symengine/tests/cwrapper/test_cwrapper.c

```diff
@@ -458,6 +458,69 @@ void test_ascii_art() {
     basic_str_free(s);
 }
 
+void test_ntheory() {
+    basic x1, x2, x3, x4, x5, x6, x7, x8,  i1, i2, i4, i5;
```

Comment:
You don't need all the variables `x1, x2, x3, x4, x5, x6, x7, x8`. Just use one `x`